rotkehlchen/api/v1/schemas.py

```diff
@@ -1779,25 +1780,20 @@ class QueriedAddressesSchema(Schema):
 
 
 class DataImportSchema(AsyncQueryArgumentSchema):
-    source = fields.String(
-        required=True,
-        validate=webargs.validate.OneOf(
-            choices=(
-                'cointracking.info',
-                'cryptocom',
-                'blockfi-transactions',
-                'blockfi-trades',
-                'nexo',
-                'shapeshift-trades',
-                'uphold',
-                'bisq',
-                'binance',
-            ),
-        ),
-    )
+    source = SerializableEnumField(enum_class=DataImportSource, required=True)
     file = FileField(required=True, allowed_extensions=('.csv',))
     timestamp_format = fields.String(load_default=None)
 
+    @post_load
+    def transform_data(  # pylint: disable=no-self-use
+            self,
+            data: Dict[str, Any],
+            **_kwargs: Any,
+    ) -> Any:
+        if data['timestamp_format'] is None:
```

Comment:
why do this?

Reply:
Further in the importers we are now using default values for `timestamp_format`. And if frontend passes `None`, then `None` is used instead of the default value. And thus we remove this argument.

Reply:
Why remove it here and not treat None further down the line? I don't understand the logic here.

Reply:
As I said in discord can you put  a comment here explaining why you pop it if it;s None? Basically so that the default value of the argument in each function kicks in.

And a request here. If the default value of the timestamp format argument is the same in every importer function remove default arguments and add the default functionality here. So if it's `None` then change it to the default value here and propagate it downwards.